doc/env-settings.md

```diff
@@ -32,6 +32,13 @@ Public URL of [Layman Test Client](https://github.com/LayerManager/layman-test-c
 ### LAYMAN_TIMGEN_URL
 Internal URL of thumnbail image generator (Timgen) used for generating map thumbnails.
 
+### LAYMAN_INPUT_SRS_LIST
+List of [EPSG codes](https://en.wikipedia.org/wiki/EPSG_Geodetic_Parameter_Dataset) that are accepted as native for layers and map compositions. Value consists of integer codes separated by comma (`,`). If the list does not contain codes [4326](https://epsg.io/4326) and [3857](https://epsg.io/3857), they are appended by Layman automatically.
+Only subset of these codes is allowed: `3857,4326,5514,32633,32634,3034,3035,3059`
+   - Sample SRS list for World: `4326,3857`
+   - Sample SRS list for Czech Republic: `4326,3857,5514,102067,32633,32634`
+   - Sample SRS list for Latvia: `4326,3857,3059`
+
 ### LAYMAN_OUTPUT_SRS_LIST
 List of [EPSG codes](https://en.wikipedia.org/wiki/EPSG_Geodetic_Parameter_Dataset) that will be supported as output spatial reference systems in both WMS and WFS. Value consists of integer codes separated by comma (`,`). If the list does not contain codes [4326](https://epsg.io/4326) and [3857](https://epsg.io/3857), they are appended by Layman automatically.
```

Comment:
`[4326](https://epsg.io/4326) and [3857](https://epsg.io/3857)` -> `from [LAYMAN_INPUT_SRS_LIST](#LAYMAN_INPUT_SRS_LIST)`